source/create-agg-pipeline.txt

```diff
@@ -56,6 +56,15 @@ collection.
    to perform :atlas:`full-text search</atlas-search/atlas-search-overview>` 
    on Atlas collections.
 
+Required Roles 
+--------------
+
+To create and execute aggregation pipelines for an Atlas cluster, you must have 
+the :authrole:`Project Data Access Read Only` role or higher.
```

Comment:
Consider "... Read Only` role or a role with more privileges" or similar, instead of "or higher".